advanced/index.md

```diff
@@ -49,8 +49,8 @@ Explore the structure and way change is preserved in Git.
 ---
 
 {% capture lab %}
-1. Understand the way Git preserves history and data objects
-2. Utilize shorthand for navigation commit history
+1. Understand the way Git preserves data
```

Comment:
The history of that data, where it's been and what it was at certain points in time, is one of the fundamental aspects of Git. It's not just about preserving data, but tracking its evolution.